spec/models/user/oauth2_data_spec.rb

```diff
@@ -31,14 +31,14 @@
     tokens = User::Oauth2Data.get(user_id, app_id)
     expect(tokens[:access_token]).to eq access_token
     expect(tokens[:refresh_token]).to eq refresh_token
-    expect(tokens[:expiration_time]).to eq 1
+    expect(tokens[:uc_clc_expire]).to eq 1
     expect(Cache::UserCacheExpiry).to receive(:notify).once
     updated_access_token = random_string 10
     User::Oauth2Data.new_or_update(user_id, app_id, updated_access_token)
     updated_tokens = User::Oauth2Data.get(user_id, app_id)
     expect(updated_tokens[:access_token]).to eq updated_access_token
-    expect(updated_tokens[:refresh_token]).to be_nil
-    expect(updated_tokens[:expiration_time]).to be_nil
+    expect(updated_tokens[:refresh_token]).to be_empty
```

Comment:
@frankiepun - Ideally we want to make sure that the ActiveRecord models behave like they did before the migration. I recall that the reason you're adjusting this test is to ensure that it passes if it returns an empty string (`''`) rather than an actual `NilClass` object.

Could we perhaps apply the mixin approach from [Make Values Nil If Blank & Data Normalization In Rails](https://solidfoundationwebdev.com/blog/posts/make-values-nil-if-blank-data-normalization-in-rails). Might want to wrap the logic so that it only applies if the current database is using Oracle Enhanced adapter.

Reply:
Good point. I don't think any callers are accessing these fields directly. So we should be fine.

Reply:
We will leave this alone for now and change if we discover any issues during end-to-end testing.